site/docs/pages/index.mdx

```diff
@@ -288,173 +386,154 @@ setFilteredTokens(filteredTokens);
+        <div className="pt-[20px] xl:pt-[40px]">
+          <h2 className="text-4xl md:text-9xl text-center md:text-left">OnchainKit</h2>
+          <p className="leading-7 text-xl text-gray-400 px-0 py-6 max-w-[500px] md:text-left text-center">
+            Build your onchain apps with ready-to-use React components and Typescript utilities.
+          </p>
+          <a id="alternate-button" href="/getting-started" className="mx-auto md:mx-0 flex justify-center items-center bg-indigo-600 text-[white] font-bold leading-6 px-4 py-3 rounded-xl w-[180px]">
+            Get started
+          </a>
+        </div>
       </div>
-      <div className="flex flex-row items-center space-x-[128px]">
-        <ul className="flex space-x-[28px]">
-          <li>[Getting Started](/getting-started)</li>
-          <li>[Warpcast](https://warpcast.com/~/channel/onchainkit)</li>
-          <li>[GitHub](https://github.com/coinbase/onchainkit)</li>
+      <div className="flex flex-col md:flex-row items-center md:space-x-[128px]">
+        <ul className="flex space-x-[28px] py-3 md:py-0">
+          <li>[Docs](/getting-started)</li>
+          <li>[Playground](https://an-onchain-app-in-100-components.vercel.app/)</li>
+          <li>[Template](https://buildonchainapps.xyz/)</li>
+          <li>[Figma](https://www.figma.com/community/file/1370194397345450683/onchainkit)</li>
+        </ul>
+        <ul id="social-icons-list" className="flex space-x-[28px]">
+          <a href="https://github.com/coinbase/onchainkit" target="_blank">
+            <svg width="20" height="19" viewBox="0 0 20 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path fillRule="evenodd" clipRule="evenodd" d="M10.0719 0.19519C4.83504 0.19519 0.601562 4.45988 0.601562 9.73589C0.601562 13.9533 3.3141 17.5232 7.07711 18.7867C7.54758 18.8817 7.71991 18.5814 7.71991 18.3289C7.71991 18.1077 7.70441 17.3495 7.70441 16.5596C5.06999 17.1283 4.5214 15.4223 4.5214 15.4223C4.09803 14.3166 3.47073 14.0324 3.47073 14.0324C2.60849 13.4479 3.53354 13.4479 3.53354 13.4479C4.48999 13.5111 4.99187 14.4272 4.99187 14.4272C5.83841 15.8803 7.20253 15.4698 7.75132 15.217C7.82963 14.6009 8.08067 14.1745 8.34721 13.9376C6.24608 13.7164 4.03541 12.8951 4.03541 9.23033C4.03541 8.1878 4.41148 7.33487 5.00738 6.67151C4.91336 6.43463 4.58401 5.45511 5.10159 4.1441C5.10159 4.1441 5.90122 3.89132 7.70421 5.12343C8.47614 4.91459 9.27221 4.80835 10.0719 4.80745C10.8715 4.80745 11.6867 4.91814 12.4394 5.12343C14.2426 3.89132 15.0422 4.1441 15.0422 4.1441C15.5598 5.45511 15.2302 6.43463 15.1362 6.67151C15.7478 7.33487 16.1084 8.1878 16.1084 9.23033C16.1084 12.8951 13.8977 13.7005 11.7809 13.9376C12.1259 14.2377 12.4237 14.8062 12.4237 15.7066C12.4237 16.9861 12.4082 18.0129 12.4082 18.3287C12.4082 18.5814 12.5807 18.8817 13.051 18.7869C16.814 17.523 19.5265 13.9533 19.5265 9.73589C19.542 4.45988 15.293 0.19519 10.0719 0.19519Z" fill="#F9FAFB"/>
```

Comment:
@abcrane123 We really need to start creating a folder for all SVGs. it's getting a bit out of hand.